test/evergreen.yml

```diff
@@ -1582,8 +1582,8 @@ variables:
       - func: "fetch artifacts"
       - func: "format test disagg"
         vars:
-          format_args: disagg.page_log=palite disagg.mode=follower ops.verify=1 runs.mirror=1
-          times: 5
+          format_args: disagg.page_log=palite disagg.mode=follower ops.verify=1 runs.mirror=1 runs.timer=10:15
```

Comment:
is there value in one test/format run, simply running for 60-120 minutes (as longer runs is the goal for IB stability)?